apps/demos/testing/skipped-tests.js

```diff
@@ -117,7 +114,7 @@ export const skippedTests = {
       { demo: 'ContextMenu', themes: [THEME.material] },
     ],
     Scheduler: [
-      { demo: 'Overview', themes: [THEME.fluent, THEME.material] },
+      // NOTE: 'GroupByDate' demo has problems with rendering
```

Comment:
This comment is overhead. If we have a line below we already know that we have problems with rendering. There is no additional information here. Please, be careful in next PRs.